contributing.md

```diff
@@ -5,34 +5,39 @@ title: Contributing to Bazel
 
 # Contributing to Bazel
 
+There are many ways to help the Bazel project and ecosystem.
 
-## How can I help?
+## Provide feedback
 
-There are many ways to help the Bazel project and ecosystem.
+As you use Bazel, you may find things that can be improved. You can help by:
 
-1. Provide feedback as you use Bazel. As you use it, you may find things
-   that can be improved. Please report them:
-   - Report bugs. In particular, it's important to report any crash or
+   - Reporting bugs. In particular, it's important to report any crash or
      correctness bug.
-   - Report issues when the documentation is incomplete or unclear.
-   - Report issues when an error message could be improved.
+   - Reporting issues when the documentation is incomplete or unclear.
```

Comment:
Should we add a link to documentation issues? Perhaps point people at the "Create issue" link in the top right corner of the page?

Reply:
Mentioned the Create issue link